backend/src/services/log.ts

```diff
@@ -15,6 +15,14 @@ interface BunyanLog {
   src: { file: string; line: number }
   msg: string
 }
+
+interface RequestCompletedLog extends BunyanLog {
+  method: string
+  url: string
+  status: number
+  'response-time': number
```

Comment:
Is the a reason for kebab case here? Seems strange to not use camel case unless there's something I'm missing